### ISSUE TITLE
Linux issue

### ISSUE THREAD

**Issue by ghost** [dizzyup/hypest-dark#2]
Hey, theme has a weird artefact on Linux on the bottom right corner:
here is a capture:

![photo_2019-11-09_22-17-09](https://user-images.githubusercontent.com/20411249/68535251-b99b3100-033f-11ea-8869-898ec82c4d84.jpg)

**Comment by dizzyup** [comment]
@fwhcat Super strange! Thanks for reporting.

- Does it disappear if the window size changes or you switch tabs/views? 
- Do you have any plugins enabled that might be causing it?

**Comment by ghost** [comment]
Sorry for the delay @dizzyup.

1) it doesn't disappear while dragging/resizing, however it disappears when I'm in the settings page.
2) I only had minimap plugin, I tried without it, same issue.

**Comment by dizzyup** [comment]
@fwhcat No problem! That's useful info, means it's something specific to the editor views at least.

Would you mind sharing the contents of your `config.cson` file? Not exactly sure where the menu item lives on Linux but it should be named `Config...` or similar. There shouldn't be any secrets in there but please double check!

**Comment by ghost** [comment]
Here is the content of the file:

```
"*":
  core:
    disabledPackages: [
      "spell-check"
      "metrics"
    ]
    telemetryConsent: "no"
    themes: [
      "hypest-dark"
      "hypest-dark-syntax"
    ]
  "exception-reporting":
    userId: "some guid here"
  "hypest-dark":
    tabCloseButton: "Right"
    vibrancy: true
  minimap:
    absoluteMode: true
    adjustAbsoluteModeHeight: true
  welcome:
    showOnStartup: false
```

I tried with or without vibrancy option, doesn't change.

**Comment by dizzyup** [comment]
I haven't been able to track this down, @fwhcat - I'm not on Linux here which unfortunately makes it almost impossible to properly debug - I haven't been able to replicate anything like this locally here on macOS.

The only thing I can suggest is to open the developer console in Atom (https://flight-manual.atom.io/hacking-atom/sections/debugging/#check-for-errors-in-the-developer-tools), and try and find whatever that element is in the dom. If you were able to work out what the element was and what the classes/styles were applied to it I might be able to work out what's causing it.

Sorry I can't be more help!

**Comment by ghost** [comment]
Hi,

I found out with the developer console. The artefact I was seeing is : 

```html
<div class="scrollbar-corner" style="position: absolute; height: 20px; width: 20px; bottom: 0px; right: 0px; overflow: scroll;"></div>
```

**Comment by dizzyup** [comment]
@fwhcat I think I might have this addressed in `v0.8.0` which I've just published. That update adds styling for scollbars - I don't have a way to check it on Linux but it works well here on macOS.

The styling isn't enabled by default (to prevent it interfering with native styles on macOS) but if you enable the new setting for `Theme scrollbars` you should see the new styles, and I'm hoping they fix the issue you've been seeing.

<img width="878" alt="Screenshot 2019-12-02 at 13 51 28" src="https://user-images.githubusercontent.com/717867/69965030-5306ce80-150b-11ea-8d1c-70fc0f754000.png">


You may want to close any active windows and reopen them if they don't take effect at first.

**Comment by ghost** [comment]
@dizzyup Nice work it indeed fixes the issue !
